docs/.gitignore

```diff
@@ -1,2 +1,4 @@
 _site
-.jekyll-cache
\ No newline at end of file
+.jekyll-cache
+.bundle
```

Comment:
Any specific reason to add these here?

Reply:
Yes, my WSL  added that to my directory for me to be able to run the site locally. So I added it to git ignore so I won't have to push it since its specific to my machine alone.

Reply:
Sounds real good.